lib/sgn-sdk.ts

```diff
@@ -67,7 +67,13 @@ if (isBrowser()) {
 
         if (trackId) scriptConfig.eventTracker = new Tracker({trackId});
```

Comment:
You can also just skip creating an eventTracker entirely at this step, if the environment is staging.